tests/security_features_test.go

```diff
@@ -291,18 +292,24 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 				Expect(err).ToNot(HaveOccurred())
 			}
 
-			By("Re-enabling the custom policy by removing the corresponding feature gate")
-			tests.DisableFeatureGate(virtconfig.DisableCustomSELinuxPolicy)
+			if featureGateEnabledByTest {
```

Comment:
You don't need this as previous Kubevirt spec should be applied or am I missing anything?

Reply:
You mean the post-test cleanup? Yeah I guess... I try not to rely too much on it and leave things roughly the way I found them.
If you feel strongly about this let me know and I'll remove that part.